documentation/source/content.rst.inc

```diff
@@ -1,8 +1,3 @@
-=============
-Documentation
-=============
-
-.. include:: ../../README.rst
 .. include:: /faq/main.rst
```

Comment:
Remove this file, iam not sure why i used it initially but it can be removed afaik.

Reply:
Done.